src/components/input-group/demo/styles.scss

```diff
@@ -0,0 +1,6 @@
+@use "../../../mixins/ms";
+
+.demo-input-group {
+  display: grid;
+  grid-gap: ms.step(1);
+}
```

Comment:
I don't think this file would be necessary if you used a Rhythm object (see earlier comment)